lib/bolt/inventory.rb

```diff
@@ -177,6 +177,22 @@ def groups_in(node_name)
     end
     private :groups_in
 
+    def self.magic_local_config(target, data)
+      if target.name == 'localhost'
```

Comment:
nitpick. I think move this conditional out to the caller and call this "localhost_defaults"

Reply:
can we use deep merge?